dessia_common/workflow/utils.py

```diff
@@ -65,3 +69,41 @@ def nonblock_variables_to_script(nonblock_variables, prefix, imports, imports_as
         imports_as_is.extend(nbv_script.imports_as_is)
         nbvs_str += f"{prefix}variable_{nbv_index} = {nbv_script.declaration}\n"
     return nbvs_str
+
+
+def update_imports(input_type, input_name):
+    """ Update imports based on the schema of the input. """
+    schema = get_schema(annotation=input_type, attribute=SchemaAttribute(input_name))
+    return schema.get_import_names(import_names=[])
+
+
+def generate_input_string(signature: str, sequence_length: int = 1):
```

Comment:
You should still pass the sequence instead of the sequence length, as you did before.

The argument you should pass is the `('Set your arguments here')` or `.from_file('Set your filepath here')` part

Reply:
done !